### PR TITLE
Add NGO 1.1.0 documentation

### PR DESCRIPTION
**Pull request by armstrongl**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Add the NGO 1.1.0 documentation.

**Issue Number:**

### CODE REVIEW

**Review by chrispope — APPROVED** [review]

**Comment by NoelStephensUnity** [comment]
Still reading through this in-between other tasks... just letting you know I am "working on it"  :)

File: versioned_docs/version-1.0.0/advanced-topics/bufferserializer.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (NoelStephensUnity) [inline versioned_docs/version-1.0.0/advanced-topics/bufferserializer.md:1]: I can't seem to find where this document is accessible via the sidebar.
Are we referencing it somewhere else and/or did the sidebar not get updated?

File: versioned_docs/version-1.0.0/advanced-topics/inscene_parenting_player.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    private void SetPlayerParent(ulong clientId)
+    {
+        if (IsSpawned && IsServer)
+        {
+            // As long as the client (player) is in the connected clients list
+            if (NetworkManager.ConnectedClients.ContainsKey(clientId))
+            {
+                // Set the player as a child of this in-scene placed NetworkObject 
+                NetworkManager.ConnectedClients[clientId].PlayerObject.transform.parent = transform;
```
Comment (NoelStephensUnity) [inline versioned_docs/version-1.0.0/advanced-topics/inscene_parenting_player.md:53]: **This needs to be updated to this:**
```
// Set the player as a child of this in-scene placed NetworkObject
// We parent in local space by setting the WorldPositionStays value to false
NetworkManager.ConnectedClients[clientId].PlayerObject.TrySetParent(NetworkObject, false);

```

**Review by NoelStephensUnity — APPROVED** [review]
LGTM!

**Pull request merged**